### DISCUSSION TITLE
DDOS mobile/web application

### DISCUSSION THREAD

**Discussion by LiberStorm** [WeAreMahsaAmini/Core#155] [Ideas]
We can create a mobile/web app which does the DDOS attack. 
People can use it to send a bunch of request to a specified server.
We can store a JSON file here (in a repository) to specify the target domain/ip. Then we can change it dynamically. Poeple just need to open the app to start DDOS. Or maybe we can allow users to run the DDOS in the background.
What do you think?

**Comment by fightOrF** [comment]
Mobile app is out of the question because you won't be able to get it to any store and distributing it outside of the stores creates a risk of third parties using a modified version of the app for their own purposes. 

The web version needs a huge traction before having an impact assuming you find a host that is willing to keep your site up since it would be doing something illegal (DDOS)

**Reply by LiberStorm** [reply]
It makes sense.
